### PR TITLE
Remove code to start a remote query run

### PR DESCRIPTION
**Pull request by koesie10**
This removes the `runRemoteQuery` method on the remote queries manager, all code that calls it and code which is now dead.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/src/remote-queries/remote-queries-api.ts [hunk 1]
```diff
@@ -1,154 +0,0 @@
-import { EOL } from "os";
-import { Credentials } from "../common/authentication";
-import { RepositorySelection } from "./repository-selection";
-import { Repository } from "./shared/repository";
-import { RemoteQueriesResponse } from "./gh-api/remote-queries";
```
Comment (charisk) [inline extensions/ql-vscode/src/remote-queries/remote-queries-api.ts:5]: Can we also delete this?

File: extensions/ql-vscode/src/remote-queries/remote-queries-api.ts [hunk 2]
```diff
@@ -1,154 +0,0 @@
...
-import { submitRemoteQueries } from "./gh-api/gh-api-client";
```
Comment (charisk) [inline extensions/ql-vscode/src/remote-queries/remote-queries-api.ts:6]: Shall we clean up gh-api-client as well from remote query related stuff?

**Review by charisk — APPROVED** [review]
LGTM. I think we can remove some more code but appreciate we may not want to do that in this PR.

**Comment by koesie10** [comment]
@charisk Thanks for the review! We can definitely remove a lot more code, but I think it's easier to split it up in more PRs so it can be reviewed more easily. We have a few issues that are related to cleaning up all remote queries code that is remaining after removing the code specified in separate issues.

**Pull request merged**